archinstall/lib/disk/partition.py

```diff
@@ -186,7 +186,7 @@ def _call_lsblk(self) -> Dict[str, Any]:
 			try:
 				output = SysCommand(f"lsblk --json -b -o+LOG-SEC,SIZE,PTTYPE,PARTUUID,UUID,FSTYPE {self.device_path}").decode('UTF-8')
 			except SysCallError as error:
-				# It appears as if lsblk can return exit codes like 8192 to indicate something.
+				# It appears as if lsblk can return exit codes like 32 to indicate something.
 				# But it does return output in stderr so we'll try to catch it minus the message/info.
```

Comment:
The exit status of 32 is "none of specified devices found". I recommend deletion of this line, what do you think? Maybe delete the line that follows as well?

https://man.archlinux.org/man/lsblk.8#EXIT_STATUS